lib/algfld.gd

```diff
@@ -188,3 +188,45 @@ DeclareGlobalFunction("AlgExtEmbeddedPol");
 
 DeclareGlobalFunction("AlgExtSquareHensel");
 
+#############################################################################
+##
+#F  DecomPoly( <f> [:"onlyone"] )  finds ideal decompositions of rational f
+##
+##  <#GAPDoc Label="DecomPoly">
+##  <ManSection>
+##  <Func Name="DecomPoly" Arg='pol'/>
+##
+##  <Description>
+##  Let <M>f</M> be a univariate, rational, irreducible, polynomial. A
+##  pair <M>g</M>,<M>h</M> of polynomials of degree strictly
+##  smaller than that of <M>f</M>, such that <M>f(x)|g(h(x))</M> is
+##  called an ideal decomposition. In the context of field
+##  extensions, if <M>\alpha</M> is a root of <M>f</M> in a suitable extension
+##  and <M>Q</M> the field of rational numbers, such a decomposition corresponds
+##  to (proper) subfields <M>Q\lt Q(\beta)\lt Q(\alpha)</M>, where <M>g</M> is the
```

Comment:
"such a decomposition" vs. "subfields" -- shouldn't either both be in plural, or both be in singular? 
I guess technically you can argue that of course Q and Q(\alpha) are also subfields of Q(\alpha), but I still find this mildly irritating to read... But really only mildly, I think I got the gist, thanks :-)